swc/private/swc.bzl

```diff
@@ -133,26 +135,22 @@ def _impl(ctx):
             map_outs = ctx.outputs.map_outs
         else:
             map_outs = _declare_outputs(ctx, _calculate_map_outs(srcs, ctx.attr.source_maps))
-        outputs = []
-        outputs.extend(js_outs)
-        outputs.extend(map_outs)
+        output_sources = js_outs + map_outs
```

Comment:
I'd rather not overwrite `output_sources` here.

Can we delete these 3 lines (and remove the new one), then just keep `output_sources.extend(outputs)` at the end of the loop?

Reply:
> the other easy option is to define output_sources in each branch, that way we won't be redefining it. would you prefer that?

I think that would solve both of our concerns 👍

Reply:
cool, works for me!